docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,496 @@
+
+To use this feature, enable the `traceQLStreaming` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
+
+Streaming is available for both the **Search** and **TraceQL** query types, and you'll get immediate visibility of incoming traces on the results table. This smooth integration makes data exploration a breeze and speeds up decision-making.
+
+{{< video-embed src="/media/docs/grafana/data-sources/tempo-streaming-v2.mp4" >}}
+
+### Tempo Search powered by TraceQL
+
+<!-- André Pereira -->
+
+_Generally available in all editions of Grafana_
+
+The **Search** query type was replaced with a new editor powered by TraceQL. This new editor allows you to use the same query language for both Search and TraceQL queries. This change also brings a new UI that makes it easier to write queries and explore your data while using the powerful features offered by TraceQL.
+
+The previous Search interface is now deprecated and will be removed in a future release. We recommend that you start using the new editor as soon as possible and migrate existing dashboards.
+
+{{< figure src="/media/docs/grafana/data-sources/tempo-search.png" max-width="750px" caption="Tempo Search editor powered by TraceQL" >}}
+
+### Span filtering for traces is GA
+
+_Generally available in all editions of Grafana_
+
+<!-- Joey Tawadrous -->
+
+Since the last release, we've made several improvements to span filtering. Now, we're promoting span filters out of public preview and into general availability.
+
+Span filters allow you to work much more efficiently with traces that consist of a large number of spans. Span filters exist above the trace view and allow you to filter the spans that are shown in the trace view. The more filters you add, the more specific the filtered spans.
+
+Currently, you can add one or more of the following filters:
+
+- Service Name
+- Span Name
+- Duration
+- Tags (which includes tags, process tags, and log fields)
+
+To only show the spans you've matched, you can enable the **Show matches only** toggle.
+
+Learn more about span filtering in our [Tempo data source documentation]({{< relref "../datasources/tempo/#span-filters" >}}).
+
+{{< figure src="/media/docs/tempo/screenshot-grafana-tempo-span-filters-v10-1.png" max-width="750px" caption="Traces span filtering" >}}
+
+### Configuration page redesign for Loki and Elasticsearch
+
+<!-- Matías Wenceslao Chomicki -->
+
+_Generally available in all editions of Grafana_
+
+The Loki and Elasticsearch data source configuration pages have been redesigned to make getting started and setting up data sources as simple and easy to understand as possible. You can now find new subsections with links to configuration pages, as well as tooltips to assist you with configuring and customizing data sources.
```

Comment:
@matiasb - In the second sentence, it sounds like the configuration pages in Grafana for these data sources have new subsections with links to configuration pages. Where are these other configuration pages? Or are the subsections and links not on the data source configuration pages. 

**Note, please don't commit any changes without checking in with Quynton first.**

Reply:
Oh my bad. It's actually "with links to documentation pages".